oteapi/strategies/download/sftp.py

```diff
@@ -1,21 +1,19 @@
 """Strategy class for sftp/ftp"""
 # pylint: disable=unused-argument
```

Comment:
better to remove unused arguments than to disable to warning

Reply:
Indeed. But I believe this is for the `session` parameter, which I believe we cannot remove? :/